tests/unit/FormulaTest.ts

```diff
@@ -247,6 +247,85 @@ describe('CustomFormula', () => {
             const result = compute('Report with type after 4 spaces   {report:type}-and no space after computed part', mockContext);
             expect(result).toBe('Report with type after 4 spaces   Expense Report-and no space after computed part');
         });
+
+        describe('Reimbursable Amount', () => {
+            const reimbursableContext: FormulaContext = {
+                report: {
+                    reportID: '123',
+                    reportName: '',
+                    type: 'expense',
+                    policyID: 'policy1',
+                },
+                policy: {
+                    name: 'Test Policy',
+                } as Policy,
+            };
+
+            beforeEach(() => {
+                jest.clearAllMocks();
+            });
+
+            test('should compute reimbursable amount', () => {
+                reimbursableContext.report.currency = 'USD';
+                reimbursableContext.report.total = -10000; // -$100.00
+                reimbursableContext.report.nonReimbursableTotal = -2500; // -$25.00
+                // reimbursableSpend = $100.00 - $25.00 = $75.00
+                const result = compute('{report:reimbursable}', reimbursableContext);
+                expect(result).toBe('$75.00');
+            });
+
+            test('should compute reimbursable amount with different currency', () => {
+                reimbursableContext.report.currency = 'EUR';
+                reimbursableContext.report.total = -8000; // -€80.00
+                reimbursableContext.report.nonReimbursableTotal = -3000; // -€30.00
+                // reimbursableSpend = €80.00 - €30.00 = €50.00
+                const result = compute('{report:reimbursable}', reimbursableContext);
+                // TODO: Update when `formatAmount()` supports the EUR symbol; it should be '€50.00'
```

Comment:
I'm confused, we clearly display the euro symbol in the app already, so why can't we here? Also, below is proof that we use the symbol and not EUR on OldDot. Please fix this.

<img width="662" height="105" alt="Image" src="https://github.com/user-attachments/assets/e8c64b40-50ce-49e6-b3e1-02c377f7836a" />